### PR TITLE
Switch to in-code styling for consistency

### PR DESCRIPTION
**Pull request by sarodz**
* ran the prettier formatter as well

### CODE REVIEW

File: src/components/common/modal.ts
Comment (sarodz) [inline src/components/common/modal.ts:1]: Important change here

File: src/components/policy-modal/policyModal.tsx [hunk 1]
```diff
@@ -1,87 +1,83 @@
...
-import SubwayPolicyScreen from "./policy-screens/subwayPolicyScreen";
-
-
+import { Button, MobileStepper, Modal, Typography } from '@mui/material';
+import { displayedPolicy, isPolicyModalOpen } from '../../selectors';
+import { closePolicyModal, incrementDisplayedPolicy, decrementDisplayedPolicy } from '../../reducers/policySlice';
+import { useAppDispatch, useAppSelector } from '../../store';
+import { KeyboardArrowLeft, KeyboardArrowRight } from '@mui/icons-material';
+import SubwayPolicyScreen from './subwayPolicyScreen';
+import { ModalBox } from '../common/modal';
```
Comment (sarodz) [inline src/components/policy-modal/policyModal.tsx:7]: the shared modal is imported here

File: src/components/appBar/styledComponent.ts
Comment (sarodz) [inline src/components/appBar/styledComponent.ts:1]: Started adding styledComponent file to each component to contain the component specific styles

File: src/theme.ts
Comment (sarodz) [inline src/theme.ts:1]: Now we apply colour across all components

File: src/App.tsx [hunk 2]
```diff
@@ -28,20 +29,23 @@ export function App() {
     dispatch(fetchZoneData());
   });
 
-
   if (isLoading) {
     return (
       <Main>
-        <CircularProgress id="initial-loader" />
+        <ThemeProvider theme={appTheme}>
```
Comment (sarodz) [inline src/App.tsx:35]: This is where we apply the theme

File: src/App.tsx [hunk 3]
```diff
@@ -28,20 +29,23 @@ export function App() {
...
       </Main>
     );
   }
 
   return (
     <Main>
-      <TopAppBar />
-      {renderPolicyModal ? <PolicyModal/> : null}
-      <Map />
+      <ThemeProvider theme={appTheme}>
```
Comment (sarodz) [inline src/App.tsx:44]: This is where we apply the theme

File: src/components/common/modal.ts
Comment (sarodz) [reply src/components/common/modal.ts:1]: We can follow this pattern to share styles across components

File: src/components/policy-modal/policyModal.tsx (hunk 1 above)
Comment (sarodz) [reply src/components/policy-modal/policyModal.tsx:7]: This is imported in the submit component as well

**Review by RevRocky — APPROVED** [review]

**Pull request merged**